libretro/Makefile.common

```diff
@@ -25,3 +25,11 @@ SOURCES_C := \
 	$(CORE_DIR)/src/ymfm/ymfm_opn.c \
 	$(CORE_DIR)/src/ymfm/ymfm_ssg.c \
 	$(CORE_DIR)/src/z80/z80.c
+    $(CORE_DIR)/libretro/libretro-common/compat/compat_posix_string.c
+    $(CORE_DIR)/libretro/libretro-common/compat/compat_strl.c
+    $(CORE_DIR)/libretro/libretro-common/compat/fopen_utf8.c
+    $(CORE_DIR)/libretro/libretro-common/encodings/encoding_utf.c
+    $(CORE_DIR)/libretro/libretro-common/streams/file_stream.c
+    $(CORE_DIR)/libretro/libretro-common/streams/file_stream_transforms.c
+    $(CORE_DIR)/libretro/libretro-common/string/stdstring.c
+    $(CORE_DIR)/libretro/libretro-common/vfs/vfs_implementation.c
```

Comment:
Inconsistent indentation, and you will also need to add a \ to everything but the last file in the list.

Reply:
Fixed